samples/map/double_sequence/CMakeLists.txt

```diff
@@ -1,5 +1,3 @@
-set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}" )
-
 add_executable( double_sequence main.cpp )
 
 target_link_libraries( double_sequence pthread tbb ${Boost_LIBRARIES} )
```

Comment:
Instead of pthread, we can use the ${CMAKE_THREAD_LIBS_INIT}.

For tbb what about TBB_LIBRARIES